src/frontend/context/AppContext.tsx

```diff
@@ -105,6 +107,21 @@ const [AppProvider, useApp] = createUseContext(({ autentisertSaksbehandler }: IP
         }
     }, [autentisertSaksbehandler]);
 
+    useEffect(() => {
+        axiosRequest<IToggles, string[]>({
+            method: 'POST',
+            url: '/familie-ba-sak/api/feature',
+            data: Object.values(ToggleNavn),
+            påvirkerSystemLaster: true,
+        }).then((response: Ressurs<IToggles>) => {
+            if (response.status === RessursStatus.SUKSESS) {
+                settToggles(response.data);
+            } else {
+                settToggles(alleTogglerAv);
+            }
+        });
+    }, []);
+
```

Comment:
Jeg kan ikke så mye om feature toggles egentlig. Men det er frontend som skal ha ansvar for å sette si at en feature skal være på? ikke at den bare henter fra backend?

Reply:
Den henter fra backend, men vi definerer hvilke toggler vi ønsker å hente frontend 👍

Reply:
ah, skjønner. Tror jeg ble litt forvirra av at det var en post, og ikke get.

Reply:
Ikke noe problem, vi bruker bare POST fordi vi trenger å sende med hvilke toggler vi ønsker status på :)